src/net.h

```diff
@@ -664,25 +684,23 @@ class TransportDeserializer {
         data_hash.SetNull();
         hasher.Reset();
     }
-
-    bool complete() const
+    bool Complete() const
```

Comment:
In commit "Use adapter pattern for the network deserializer" (efecb74677222f6c70adf7f860c315f430d39ec4)

Would be good to add `override` keyword to Complete, Read, and other overridden methods for clarity and compile errors if there are changes to superclass and subclass methods no longer match up.